shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -432,6 +437,10 @@ export function openXhr(method: string, urlString: string, withCredentials?: boo
         // cause the request to fail and we no telemetry would be sent
         _wrapSetXhrProp(xhr, strDisabledPropertyName, disabled);
     }
+    
+    if (!enableIntTracking && intExcludeTrackingEndpoints.indexOf(urlString) > -1) {
+        _wrapSetXhrProp(xhr, strDisabledPropertyName, true);
```

Comment:
This is already happening on line 438 above, so we would just need to pass in "disabled" from the caller rather than a new `enableIntTracking` parameter here.

Reply:
remain the openxhr and sender untouched, all the changes are moved into cfgsync plugin